README.md

```diff
@@ -1,2 +1,60 @@
+
+Thanks to the power of Discord, many digital communities are made between friends and similarly-minded gamers. This allows this game to be very accessible to individuals, groups of friends, or "hardcore" players who might want to team up together to tackle the hardest challenges the game has to offer.
+
+### Game Loop
+
+Players control an adventuring company of customizable characters. They venture off alone or with a party of other players to fight monsters.
+
+Combat is turn-based, with various mechanics including cooldowns and skill variety (think **Pokémon!**) on both the players' and enemy's part, creating a situation where fights should be pre-planned and tactical decisions being made during a fight.
+
+Once a monster is slain, the participating players gain monster-specific crafting material in which they use to craft new gear, be it armours or weapons.
+
+With new gear worn and in hand, players venture off to take on another battle!
+
+## Stacks Used
+
+This game is being developed using **Node.js** and **PostgreSQL**. The game is played over **Discord**, and it takes advantage of the npm package `discord.js` to allow the game's bot to describe the game world and interact with players.
+
+## Contributors
+
+Made by two friends!
+
+- [TerrenceJChan](https://github.com/TerrenceJChan) / Code Monkey, "Ideas Guy", "Hey Cam, why isn't this working?"
+- [foxfriends](https://github.com/foxfriends) / Database Guru, Houses the biggest brain, Is more interested in the code than the game ¯\_(ツ)_/¯
+
+## Progress
+
+**June 28:** This game is currently "working" in its Alpha phase in our development server! We can fight monsters using a variety of attacks, and the monsters return in kind! Focusing on creating the database to store player information for game persistence.
+
+## Roadmap
+
+| Alpha (Current)                                              | Closed-Beta (ETA Mid-Late July)                              | Open-Beta (TBD)                                              | Launch                                                       |
+| :----------------------------------------------------------- | :----------------------------------------------------------- | :----------------------------------------------------------- | ------------------------------------------------------------ |
+| Crafting (recipes, item stats, inventory)<br /><br />Fighting monsters (cooldowns, skills)<br /><br />Discord game user account creation<br /><br />Uploading and retrieving player information onto database | Closed Discord server for initial player testing<br /><br />Party features (group combat)<br /><br />Item enhancement and upgrading<br /><br />Initial power-curving design | Open Discord server for larger-scale testing<br /><br />Enemy AI for more engaging fights<br /><br />Power-curving refinement and general balancing | Game Bot available for users to add to their servers<br /><br />Playing the game ourselves! |
```

Comment:
Roadmap usually done with checklist, then you can see exactly which things are done. Easier to write than a mega table anyway, just won't be horizontal. Up to you though

Reply:
Good call, I'll add it for the next iteration!